samples/snippets/csharp/tour/arrays/arrays.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <DebugType>portable</DebugType>
```

Comment:
@mairaw this is slightly different, here we are saying we want the PDBs to be of type "portable", not the app itself. Also, in csproj land it is still required to specify RIDs in the project for SCDs.